package/gluon-mesh-vpn-core/luasrc/lib/gluon/upgrade/500-mesh-vpn

```diff
@@ -88,3 +108,16 @@ else
 	uci:delete('tunneldigger', 'mesh_vpn')
 end
 uci:save('tunneldigger')
+
+if has_wireguard then
+	uci:set('wireguard', 'mesh_vpn', 'enabled', enabled)
+	uci:foreach('wireguard', 'peer', function(peer)
+		if site.mesh_vpn.bandwidth_limit.enabled(false) then
+			uci:set('simple-tc', peer.ifname, 'limit_ingress', 0)
```

Comment:
wireguard uses multiple interfaces. the limits won't work as expected because a limit can only be enforced per-interface. The expectation however is "if I limit to 12Mbit, then the FF-Node won'T use more than 12Mbit of my internet connection". Given multiple wireguard interfaces it could be using any multiple of 12Mbit for this example.